.github/ISSUE_TEMPLATE/submodule-updates.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Superproject instructions for multiple concurrent submodule updates
+about: List all submodule PRs and provide special instructions, if any
+title: "[SUBMODULE UPDATES]"
+labels: "submodule update instructions" 
+assignees: ''
+
+---
+
+### Name and Institution (Required)
+
+Name:
+Institution:
+
+
+### Brief Description
+
+<!--- Describe your update at a high level. --->
+
+### Related Issues
+
+<!--- Describe your update at a high level. For example: --->
+
+* _none_
+
+### Submodule Pull Requests
+
+<!--- Create a list of checkboxes (using "[ ]") for your submodule pull requests. --->
+<!--- If your pull requests should be merged in a specific order, please make sure they are ordered correctly. --->
+
+* [ ] https://github.com/geoschem/geos-chem/pull/834 
```

Comment:
Should we make example numbers like pull/xxxxand note that we should replace then xxxx with the issue number?